contracts/Staking.sol

```diff
@@ -19,8 +19,8 @@ contract Staking is Ownable, ReentrancyGuard {
     using SafeERC20 for IERC20;
 
     enum Position {
```

Comment:
I get the Idea to move away from trading specific semantic but as you know i'm not a fan of the OK/KO pair as it is removing the notion of temporality (expected future result)

Reply:
I agree, but that's exactly what we need to remove here :)